google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/project/DefaultAppEngineProjectService.java

```diff
@@ -332,23 +334,41 @@ String getServiceNameFromAppEngineWebXml(XmlFile appengineWebXml) {
 
   @Override
   public void generateAppYaml(FlexibleRuntime runtime, Module module, Path outputFolderPath) {
+    UsageTrackerProvider.getInstance()
+        .trackEvent(GctTracking.APP_ENGINE_FLEX_APP_YAML_CREATE)
```

Comment:
What do you think of distinguishing between the different ways to create app.yaml:

- via the checkbox when framework support is added
- via the generate button in the facet

We could enhance this later too